### PR TITLE
test/OWNERS: add pohly as approver

### PR DESCRIPTION
**Pull request by pohly**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

As technical lead in SIG Testing it would be useful to have the right to approve test changes...

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

/assign @aojea

### CODE REVIEW

**Comment by aojea** [comment]
I think we should just add ourselves to the corresponding aliases https://github.com/kubernetes/kubernetes/blob/1edbb8cf1a864c2dd71a1e5b7282de4bc3d8f5fa/OWNERS_ALIASES#L214-L222 

sig-testing-reviewers and sig-testing-approvers will give you the same rights (well earned)

**Comment by pohly** [comment]
`sig-testing-reviewers` isn't used for `test`, so I added myself there as before. For approval rights I modified the alias and also cleaned up a bit while at it.

File: test/OWNERS [hunk 1]
```diff
@@ -10,20 +10,17 @@ reviewers:
...
 approvers:
   - andrewsykim
-  - aojea
-  - BenTheElder
   - bowei # for test/e2e/{dns*,network}.go
   - caseydavenport # for test/e2e/{dns*,network}.go
-  - cblecker
   - dchen1107
   - deads2k
-  - dims
```
Comment (dims) [inline test/OWNERS:26]: Am i losing my privileges @pohly ?

Comment (pohly) [reply test/OWNERS:26]: Oops, yes, you are.

I thought you were in sig-testing-approvers, but you aren't. That was only sig-testing-reviewers.

Fixed.

Comment (dims) [reply test/OWNERS:26]: > Oops, yes, you are.

thanks @pohly !

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -212,13 +212,17 @@ aliases:
...
     - soltysh
   sig-testing-reviewers:
+    - aojea
     - bentheelder
     - cblecker
-    - spiffxp
     - dims
+    - pohly
+    - spiffxp
   sig-testing-approvers:
```
Comment (BenTheElder) [inline OWNERS_ALIASES:221]: I think we should deprecate this alias instead, it only has a few people and we don't need to apply it to many directories.

Comment (BenTheElder) [reply OWNERS_ALIASES:221]: (And aliases obscure ownership)

Comment (pohly) [reply OWNERS_ALIASES]: I agree and removed it in the first commit of this PR, then added me in the second.

Comment (pohly) [reply OWNERS_ALIASES]: @dims: do you agree? Can you re-add the LGTM and perhaps approval?

**Review by BenTheElder — COMMENTED** [review]
/lgtm
/approve
/priority important-longterm
/triage accepted

**Comment by BenTheElder** [comment]
/hold cancel

Thanks for all your work so far and thanks for cleaning that alias up!

**Comment by aojea** [comment]
> /lgtm

👏

**Pull request merged**